### ISSUE TITLE
Investigate Missing Aria Warning

### ISSUE THREAD

**Issue by james-rae** [fgpv-vpgf/fgpv-vpgf#3047]
In IE, when identify results for multiple features are shown in the results pane, the console will spit out the following line once for each result

`ARIA: Attribute " aria-label ", required for accessibility, is missing on node: [object HTMLButtonElement]`

It doesn't give any more information as to which button is the problem (e.g. the result list item itself, the magnifying glass, the chevron icon).

Investigate to see if this is a real accessibility problem, or if it's just IE being IE.
